### ISSUE TITLE
dyld: Symbol not found: _$SBOWV

### ISSUE THREAD

**Issue by MaksOta** [Flinesoft/BartyCrouch#126]
After the recent update to XCODE to Version 10.2 the script fails to run. Here is the report:

```
dyld: Symbol not found: _$SBOWV
  Referenced from: /usr/local/Cellar/bartycrouch/4.0.0/lib/libSwiftSyntax.dylib
  Expected in: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx/libswiftCore.dylib
 in /usr/local/Cellar/bartycrouch/4.0.0/lib/libSwiftSyntax.dylib
trap: 6           bartycrouch update -x
dyld: Symbol not found: _$SBOWV
  Referenced from: /usr/local/Cellar/bartycrouch/4.0.0/lib/libSwiftSyntax.dylib
  Expected in: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx/libswiftCore.dylib
 in /usr/local/Cellar/bartycrouch/4.0.0/lib/libSwiftSyntax.dylib
trap: 6           bartycrouch lint -x
Command PhaseScriptExecution failed with a nonzero exit code
```

**Comment by seasox** [comment]
See #123, fixed in 4.0.1

**Comment by MaksOta** [comment]
Thanks.

Closing this duplicate
